k8s/helm-charts/seldon-core-v2-setup/templates/seldon-v2-components.yaml

```diff
@@ -183,7 +183,7 @@ rules:
   - list
   - watch
 ---
-{{- if .Values.controller.clusterwide -}}
+{{- if and (not .Values.controller.skipClusterRoleCreation) (or .Values.controller.clusterwide .Values.controller.watchNamespaces) -}}
```

Comment:
This is the cluster role specific to the operator. Perhaps we should reflect that in the value name? Something like `skipOperatorClusterRoleCreation`